### DISCUSSION TITLE
Fork problem.

### DISCUSSION THREAD

**Discussion by taitep** [raysan5/raylib#3304] [General]
A github user (@octopusengine) has forked raylib, but modified the readme to say he made it.

**Comment by orcmid** [comment]
I think you are seeing Ramon's README as it was when the fork was done.  The fork is currently badly behind `raysan5/raylib` changes.

**Comment by raysan5** [comment]
@Taitep Please, could you provide the link to the repository?

**Comment by orcmid** [comment]
It's here: https://github.com/octopusengine/raylib

It hasn't been touched in 7 years.

**Comment by taitep** [comment]
if that matters, this discussion does not matter.

**Comment by taitep** [comment]
also they blocked issues somehow.

**Comment by taitep** [comment]
> I think you are seeing Ramon's README as it was when the fork was done.  The fork is currently badly behind `raysan5/raylib` changes.

so the readme previously looked like that?

**Reply by raysan5** [reply]
I think so
